### ISSUE TITLE
Tests für neues Feature: Output DataLoader, DataSet

### ISSUE THREAD

**Issue by AshsGit11** [Cortys/SEP-Probly-WS2526#389]
- Tests ob Daten die von FODataGenerator generiert wurden benutzt können um modelle zu trainieren
- FOData dataloader explizit getestet auf DummyModell **FERTIG**
- mathematik: angewendet/fixed und recherchiert zum erstellen und testen von trainieren von Modellen per FO Data welches generiert wurde (siehe tests: test_first_order_generator.py, KullbackLeibler, Softmax Log-Softmax, L1 Distance, (Epsilon Credal Coverage))

**Comment by AshsGit11** [comment]
siehe changes und hinzugefügte Tests zu Datei test_first_oreder_generator.py ab Zeile 82
